roq-management/src/main/java/org/roqmessaging/management/config/internal/GCMPropertyDAO.java

```diff
@@ -26,14 +26,17 @@
 	
 	//The frequency period for the update to the configuration server
 	private int period = 60000;
-	
+	// Time for the GCM to respond
+	private int hcmTIMEOUT = 5000;
 	//Define whether we format the DB
 	private boolean formatDB = false;
+	// Indicates if the master must register the cloud configuration
+	private boolean hasCloudConfiguration = false;
```

Comment:
It's usually not a good idea to have a variable with the same as a method. You (or someone else) will run into problems. Please change the variable name.